configs/webpack-config-compass/src/index.ts

```diff
@@ -0,0 +1,652 @@
+import type {
+  Compiler,
+  Configuration,
+  WebpackOptionsNormalized,
+} from 'webpack';
+import { EnvironmentPlugin } from 'webpack';
+import { merge } from 'webpack-merge';
+import fs from 'fs';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import ReactRefreshWebpackPlugin from '@pmmmwh/react-refresh-webpack-plugin';
+import path from 'path';
+import { builtinModules } from 'module';
+import type { ChildProcess } from 'child_process';
+import { spawn } from 'child_process';
+import { once } from 'events';
+
+const electronVersion = (() => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const [maj, min] = require('electron/package.json').version.split(
+    '.'
+  ) as string[];
+  return `${maj}.${min}`;
+})();
+
+const electronBinary: string = (() => {
+  return require('electron');
+})();
```

Comment:
```suggestion
const electronBinary: string = require('electron');
```

… or am I missing something? :)

Reply:
Nope, that's totally how it should've been done, I think I just poorly copied a few lines above, thanks for catching it!